sql/core/src/main/scala/org/apache/spark/sql/SQLConf.scala

```diff
@@ -36,6 +36,8 @@ private[spark] object SQLConf {
   val PARQUET_INT96_AS_TIMESTAMP = "spark.sql.parquet.int96AsTimestamp"
   val PARQUET_CACHE_METADATA = "spark.sql.parquet.cacheMetadata"
   val PARQUET_COMPRESSION = "spark.sql.parquet.compression.codec"
+  val PARQUET_BLOCK_SIZE = "spark.sql.parquet.blocksize"
+  val PARQUET_PAGE_SIZE = "spark.sql.parquet.pagesize"
```

Comment:
@yhuai  I know it will work if we create the table by external data source API, here is for making it consistency with Hive table.

Sorry I should make it clearer, and I've updated the description.